file-download.js

```diff
@@ -3,14 +3,19 @@ var React = require('react');
 module.exports = function(data, filename) {
     var blob = new Blob([data], {type: 'text/csv'});
     if (typeof window.navigator.msSaveBlob !== 'undefined') {
-            // IE workaround for "HTML7007: One or more blob URLs were revoked by closing the blob for which they were created. These URLs will no longer resolve as the data backing the URL has been freed."
-            window.navigator.msSaveBlob(blob, filename);
-        } else {
+        // IE workaround for "HTML7007: One or more blob URLs were 
+        // revoked by closing the blob for which they were created. 
+        // These URLs will no longer resolve as the data backing 
+        // the URL has been freed."
+        window.navigator.msSaveBlob(blob, filename);
+    }
+    else {
         var csvURL = window.URL.createObjectURL(blob);
         var tempLink = document.createElement('a');
         tempLink.href = csvURL;
         tempLink.setAttribute('download', filename);
-        tempLink.setAttribute('target', '_blank');
```

Comment:
Did this line break your solution? If not can you add it back? @josiahsprague added this line in PR #1 for a good reason.